### ISSUE TITLE
Dependencies cannot be resolved with fastlane 2.0

### ISSUE THREAD

**Issue by ndonald2** [HazAT/badge#37]
The dependencies for badge cannot currently be resolved against the fastlane 2.0 dependencies.

A gemfile with the following:

```
gem 'badge', '~> 0.6'
gem 'fastlane', '~> 2.0'
```

Produces the dependency error:

```
Bundler could not find compatible versions for gem "commander":
  In Gemfile:
    fastlane (~> 2.0) was resolved to 2.0.1, which depends on
      xcode-install (~> 2.0.0) was resolved to 2.0.9, which depends on
        spaceship (< 1.0.0, >= 0.25.1) was resolved to 0.39.0, which depends on
          credentials_manager (>= 0.16.0) was resolved to 0.16.2, which depends on
            commander (>= 4.3.5)

    fastlane (~> 2.0) was resolved to 2.0.1, which depends on
      commander (< 5.0.0, >= 4.4.0)

    badge (~> 0.6) was resolved to 0.6.0, which depends on
      fastlane_core (< 1.0.0, >= 0.44.0) was resolved to 0.44.0, which depends on
        commander (= 4.3.5)
Bundler could not find compatible versions for gem "rubyzip":
  In Gemfile:
    fastlane (~> 2.0) was resolved to 2.0.1, which depends on
      rubyzip (< 2.0.0, >= 1.2.0)

    badge (~> 0.6) was resolved to 0.6.0, which depends on
      fastlane_core (< 1.0.0, >= 0.44.0) was resolved to 0.50.2, which depends on
        rubyzip (~> 1.1.6)
```

**Comment by marcferna** [comment]
You can use `gem 'badge', git: 'git@github.com:HazAT/badge.git'` in your Gemfile to point to this repo until the owner releases a new version of the gem.

**Comment by tbaranes** [comment]
Si  
```
``` owls'

**Comment by tbaranes** [comment]
Iikzild_

**Comment by HazAT** [comment]
Hey ... 
Just released version `0.7.0` ... please update, the problem should be fixed
